pages/common/emulator.md

```diff
@@ -0,0 +1,36 @@
+# emulator
+
+> Manager Android emulators from the command line.
+> More information: <https://developer.android.com/studio/run/emulator-commandline>.
+
+- Display the help:
+
+`emulator -help`
+
+- Start an Android emulator device:
+
+`emulator -avd {{name}}`
+
+- Display the web cameras on your development computer that are available for emulation:
+
+`emulator -avd {{name}} -webcam-list`
+
+- Start an emulator overriding the facing back camera setting (or -camera-front for front camera):
+
+`emulator -avd {{name} -camera-back {{none|emulated|webcamN}}`
+
+- Start an emulator, with a maximun network speed:
+
+`emulator -avd {{name} -netspeed {{gsm|hscsd|gprs|edge|hsdpa|lte|evdo|full}}`
+
+- Start an emulator with network latency:
+
+`emulator -avd {{name}} -netdelay {{gsm|hscsd|gprs|edge|hsdpa|lte|evdo|none}}`
+
+- Start an emulator, making all TCP connections through a specified HTTP/HTTPS proxy:
+
+`emulator -avd {{name}} -http-proxy {{server}}:{{port}}`
```

Comment:
How does it determine whether it's HTTP or HTTPS?

Reply:
This is the help page:
```
$ emulator -help-http-proxy
the Android emulator allows you to redirect all TCP connections through
a HTTP/HTTPS proxy. this can be enabled by using the '-http-proxy <proxy>'
option, or by defining the 'http_proxy' environment variable.

<proxy> can be one of the following:

  http://<server>:<port>
  http://<username>:<password>@<server>:<port>

the 'http://' prefix can be omitted. If '-http-proxy <proxy>' is not used,
the 'http_proxy' environment variable is looked up and any value matching
the <proxy> format will be used automatically
```
How should I format this?

Reply:
Ah, I see. Perhaps a concrete example for `{{server}}:{{port}}` would help here? What about this?
```suggestion
- Start an emulator, making all TCP connections through a specified HTTP/HTTPS proxy (port number is required):

`emulator -avd {{name}} -http-proxy {{http://example.com:80}}`
```

Reply:
hey @einverne, @schneiderl wdyt?

Reply:
I agree with @sbrl here, seems clearer this way, we always prefer explicit examples if possible.

Reply:
All suggestions applied, could you please review? @mebeim